product_multi_image/__init__.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3).
+# © 2009 Sharoon Thomas Open Labs Business Solutions
+# © 2014 Serv. Tecnol. Avanzados Pedro M. Baeza
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+# © 2016 Pedro M. Baeza, Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import models
+from .hooks import pre_init_hook
```

Comment:
Cannot uninstall: `AttributeError: 'module' object has no attribute 'uninstall_hook'`. Import it too please.